### PR TITLE
Flask shows hello world

### LINKED ISSUE #5: Create the starter code for backend api

### PR DESCRIPTION
**Pull request by madelesi**
closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,71 @@
...
+  - On the left side-bar, click Extensions, search &  install `Python`
+  - Command Palette `CTRL+SHIFT+P`
+  - Type `Python: Select Interpreter`
+  - Put `.\backend\venv\Scripts\python.exe` *It should auto-complete, otherwise `Enter path...` Then `Find` and pick the right one with your mouse*
+
+- Done! if you want, make VSCode use `Git Bash` by default so you don't have to repeat that step (See [picture](vscode-list-terminals-place.png).
+- (Note: There are obviously many ways to set it up, but this is a recommended & simple way)
+</details>
+<details>
+
```
Comment (cooperemily02) [inline README.md:62]: I would add that to start the server locally, you need to write python api.py, I genuinely didn't know how to start it haha

**Review by cooperemily02 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (madelesi) [reply README.md:63]: Just clarified that, thanks for going through the trouble!

**Review by cooperemily02 — APPROVED** [review]
LGTM!

**Comment by madelesi** [comment]
TODO: Clarify the instructions more & separate Mac/Linux from Windows specific parts

**Review by egpickard — APPROVED** [review]

**Pull request merged**